torch/onnx/symbolic_opset9.py

```diff
@@ -2311,11 +2311,12 @@ def _var_mean(g, input, dim, unbiased, keepdim):
         num_elements = numel(g, input)
     else:
         mean = g.op("ReduceMean", input, axes_i=dim, keepdims_i=keepdim)
+        t_mean = g.op("ReduceMean", input, axes_i=dim, keepdims_i=1)
         redudced_dims = g.op("Shape", input)
         # dim could contain one or multiple dimensions
         redudced_dims = g.op("Gather", redudced_dims, g.op("Constant", value_t=torch.tensor(dim)), axis_i=0)
         num_elements = g.op("ReduceProd", redudced_dims, keepdims_i=0)
-    sub_v = g.op("Sub", input, mean)
+    sub_v = g.op("Sub", input, t_mean)
```

Comment:
(1) Why `t_mean` is only in `else` branch? it need also be in `if` branch, right? 
(2) if `mean` is replaced, we need delete it from `else` branch.

Reply:
Thanks, added to if branch